support-frontend/.eslintrc.js

```diff
@@ -1,10 +1,9 @@
 module.exports = {
-	extends: ['@guardian/eslint-config', 'plugin:storybook/recommended'],
```

Comment:
I was expecting to see this extend the top level eslint like the support-workers config, or is that not the intention here?